parm/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -19717,3 +19718,19 @@ surface
 ?
 ?
 ?
+IFIFIP
+4
+ncep_emc
+v2003
+local_tab_yes1
+fcst
+oper
+fcst
+fcst
+hour
+nws_ncep
+hrrr
+complex_packing_spatial_diff
+2nd_ord_sptdiff
+fltng_pnt
+lossless
```

Comment:
There are no real fields for IFI after this line. Does this intended?

Reply:
Agree with Ming's comment here--is this intentional?  Other than that, the changes look good.

Reply:
No, it was not. I'm not sure what happened, but I just regenerated the file and it looks right now.